Initial Dapr Shared Docs (#4121)
dapr/docs: daprdocs/content/en/operations/dapr-shared/_index.md

---
type: docs

> title: "Deployment strategies"

Reviewer: ```suggestion
title: "Deploy Dapr on a per-machine or per-node with Dapr Shared"
```

Reviewer: This whole topic should be moved under the Kubernetes section here https://docs.dapr.io/operations/hosting/kubernetes/ after the "Deploy Dapr on a Kubernetes cluster" topic (meaning update the index file). Also update the "Deploy Dapr on a Kubernetes cluster" topic

> title: "Deployment Strategies, Dapr Shared"

Reviewer 2: ```suggestion
title: "Deployment strategies"
```

Reviewer 2: making this entry more generic in case there are more deployment strategy options in the future besides Dapr Shared

> linkTitle: "Deployment strategies"

Reviewer: ```suggestion
linkTitle: "Dapr Shared"
```

> linkTitle: "Dapr Shared"

Reviewer 2: ```suggestion
linkTitle: "Deployment strategies"
```

weight: 100

> description: "Using different deployment strategies with Dapr Shared"

Reviewer 2: ```suggestion
description: "Learn the different deployment strategies with Dapr Shared"
```
